ONSdigital/dp-frontend-filter-dataset-controller
# API to frontend model

### What

- Added mappings from the api response model to the frontend model
- Removed any reference to CMD in the code
- Updated URLs from `/jobs/{jobID}` to `/filters/{filterID}`

### How to review

- Checkout feature/api-to-frontend-model on dp-frontend-filter-dataset-controller, dp-frontend-dataset-controller, dp-frontend-renderer, dp-frontend-router
- Run the services and visit http://localhost:20000/filters/123456/dimensions
- Verify the page continues to work as expected

### Who can review
Anyone
